Colors/Pallet/ImageProcessor.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PalletLib
+{
+    public class ImageProcessor
+    {
```

Comment:
add constructor(ctor) and make methods 'non static' its better for UnitTesting (UT)